show/main.py

```diff
@@ -2139,6 +2139,103 @@ def suppress_pending_fib(db):
     click.echo(state)
 
 
+# asic-sdk-health-event subcommand ("show asic-sdk-health-event")
+@cli.group(cls=clicommon.AliasedGroup)
+def asic_sdk_health_event():
+    """"""
+    pass
+
+
+@asic_sdk_health_event.command()
+@clicommon.pass_db
+@click.option('--namespace', '-n', 'namespace', default=None, show_default=True,
+              type=click.Choice(multi_asic_util.multi_asic_ns_choices()), help='Namespace name or all')
+def suppress_configuration(db, namespace):
+    """ Show the suppress configuration """
+    if multi_asic.get_num_asics() > 1:
+        namespace_list = multi_asic.get_namespaces_from_linux()
+        masic = True
+    else:
+        namespace_list = [multi_asic.DEFAULT_NAMESPACE]
+        masic = False
+
+    header = ['Severity', 'Suppressed category-list', "Max events"]
+    body = []
+
+    supported = False
+
+    for ns in namespace_list:
+        if namespace and namespace != ns:
+            continue
+
+        state_db = db.db_clients[ns]
+        if "true" != state_db.get(db.db.STATE_DB, "SWITCH_CAPABILITY|switch", "ASIC_SDK_HEALTH_EVENT"):
+            continue
+
+        supported = True
+
+        if masic:
+            click.echo("{}:".format(ns));
```

Comment:
This would print things like:
```
<ns1>
<ns2>
...
<table>
```

Is it expected?

Reply:
nice catch. fixed.